doc/reference/cmdline.rst

```diff
@@ -551,8 +555,34 @@ Scaffolding is available via the :command:`odoo-bin scaffold` subcommand.
     a template directory, files are passed through jinja2_ then copied to
     the ``destination`` directory
 
+
+This will create module *my_module* in directory */addons/*.
+
+.. _reference/cmdline/populate:
+
+Database Population
+===================
+
+.. program:: odoo-bin populate
+
+Odoo CLI also supports database population features.  If the feature is implemented on a given model
+(See :ref:`Populate API <reference/testing/populate/methods>`), it allows automatic data
+generation of the model's records to easily test your modules in heavier or more detailed databases.
```

Comment:
```suggestion
generation of the model's records to easily test your modules in heavily loaded databases.
```

Reply:
"heavily loaded"? Or simply "in databases containing more records"